addon/core/say-controller.ts

```diff
@@ -96,11 +97,12 @@ export default class SayController {
     return command(view.state);
   }
 
-  isMarkActive(markType: MarkType) {
+  isMarkActive(markType: MarkType, attrs?: Attrs) {
     const state = this.activeEditorState;
     const { from, $from, to, empty } = state.selection;
     if (empty) {
-      return !!markType.isInSet(state.storedMarks || $from.marks());
+      const mark = markType.isInSet(state.storedMarks || $from.marks());
+      return !!mark && (!attrs || shallowEqual(attrs, mark.attrs));
     } else {
       return rangeHasMarkEverywhere(state.doc, from, to, markType);
```

Comment:
In this case, we could loop over all the ranges of the selection, as well as pass the optional `attrs` object to `rangeHasMarkEverywhere`

Reply:
It will fixed the status of the bold toggle when having multiple table cells selected in which not all of them are bold.

Reply:
Oops, yes. Fixed now.